course_content/notebooks/dask_intro.ipynb

```diff
@@ -728,13 +728,364 @@
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/wind_speed_lake_victoria.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/E1.2098.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/atlantic_profiles.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/ostia_monthly.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/air_temp.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/pre-industrial.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/E1_north_america.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/A1B.2098.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/space_weather.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/uk_hires.pp',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/orca2_votemper.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/hybrid_height.nc',\n",
+       " '/project/avd/iris/resources/git/iris-sample-data/sample_data/SOI_Darwin.nc']"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "filepath = '/project/avd/iris/resources/git/iris-sample-data/sample_data'\n",
+    "files = glob.glob(os.path.join(filepath, '*.??'))\n",
+    "files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1. Constraining by Phenomenon"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One way to constrain data is based on the phenomenon it represents. Below we have added a constraint so only cubes with the name 'air_potential_temperature' will be loaded."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "phenom_cubes = iris.load(files, 'air_potential_temperature')\n",
+    "phenom_cubes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to achieve the same thing using dask a few steps are involved. First we create a list of constraints (the same constraint repeated) that can be fed into a map. This list must be the same length as the number of files because the function mapping will iterate through them both."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "phen_cstr = ['air_potential_temperature'] * len(files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then create a dask bag using the from_delayed method. Every file in files is passed to delayed(iris.load) along with its matching constraint from phen_cstr."
```

Comment:
Again, could we have code elements in back-ticks so that they get rendered as preformatted text please.